README.rst

```diff
@@ -208,6 +208,21 @@ To run specific tests, you can specify the test name too, something like:
 
 The ``--nocapture`` argument can be useful to see some output that otherwise is captured by nosetests.
 
+Also, you can run tests against an instance of Artifactory. Those tests should add the attribute
+``artifactory_ready``. Some environment variables have to be defined to run them. For example, for an
+Artifactory instance that is running on the localhost with default user and password configured, the
+variables could take the values:
+
+.. code-block:: bash
+
+    $ export CONAN_TEST_WITH_ARTIFACTORY=1
+    $ export ARTIFACTORY_DEFAULT_URL=http://localhost:8081/artifactory
```

Comment:
It should be clear what URL is it. I guess it is one from a repo, from the SetMeUp? It doesn't look like it

Reply:
This is not the url for the repo but the base url, the tests will create repos inside this artifactory instance to run. Maybe I can add a note about that

Reply:
Understood. As you see, not really necessary, it is correct, it was just me with some false assumptions.